### ISSUE TITLE
Netlify Deploying Error: Cannot read property '__html'

### ISSUE THREAD

**Issue by MrRobotjs** [jafaircl/gatsby-plugin-amp#9]
```
9:57:56 PM: Build ready to start
9:57:58 PM: build-image version: 84aca9ba39e0ee86ba194760fbfc51a808f62543
9:57:58 PM: buildbot version: 1ac64ca11e029436ed45ac81a38b9839778ec314
9:57:58 PM: Fetching cached dependencies
9:57:58 PM: Starting to download cache of 194.1MB
9:57:59 PM: Finished downloading cache in 1.234613067s
9:57:59 PM: Starting to extract cache
9:58:05 PM: Finished extracting cache in 5.564049919s
9:58:05 PM: Finished fetching cache in 6.909738077s
9:58:05 PM: Starting to prepare the repo for build
9:58:05 PM: Preparing Git Reference refs/heads/master
9:58:06 PM: Starting build script
9:58:06 PM: Installing dependencies
9:58:07 PM: Started restoring cached node version
9:58:09 PM: Finished restoring cached node version
9:58:09 PM: v8.15.0 is already installed.
9:58:10 PM: Now using node v8.15.0 (npm v6.4.1)
9:58:10 PM: Attempting ruby version 2.3.6, read from environment
9:58:11 PM: Using ruby version 2.3.6
9:58:12 PM: Using PHP version 5.6
9:58:12 PM: Started restoring cached node modules
9:58:12 PM: Finished restoring cached node modules
9:58:12 PM: Installing NPM modules using NPM version 6.4.1
9:58:41 PM: npm
9:58:41 PM:  WARN react-addons-clone-with-props@0.14.8 requires a peer of react@^0.14.8 but none is installed. You must install peer dependencies yourself.
9:58:41 PM: npm WARN slate-edit-list@0.11.3 requires a peer of slate@^0.32.0 but none is installed. You must install peer dependencies yourself.
9:58:41 PM: npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
9:58:41 PM: npm WARN notsup
9:58:41 PM:  SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
9:58:41 PM: added 39 packages from 67 contributors and audited 37058 packages in 28.419s
9:58:41 PM: found 6 vulnerabilities (5 moderate, 1 high)
9:58:41 PM:   run `npm audit fix` to fix them, or `npm audit` for details
9:58:41 PM: NPM modules installed
9:58:41 PM: Started restoring cached go cache
9:58:41 PM: Finished restoring cached go cache
9:58:41 PM: unset GOOS;
9:58:41 PM: unset GOARCH;
9:58:41 PM: export GOROOT='/opt/buildhome/.gimme/versions/go1.10.linux.amd64';
9:58:41 PM: export PATH="/opt/buildhome/.gimme/versions/go1.10.linux.amd64/bin:${PATH}";
9:58:41 PM: go version >&2;
9:58:41 PM: export GIMME_ENV='/opt/buildhome/.gimme/env/go1.10.linux.amd64.env';
9:58:41 PM: go version go1.10 linux/amd64
9:58:41 PM: Installing missing commands
9:58:41 PM: Verify run directory
9:58:41 PM: Executing user command: gatsby build
9:58:44 PM: success open and validate gatsby-configs — 0.027 s
9:58:45 PM: success load plugins — 0.547 s
9:58:45 PM: Found 557 cached files for public directory with 0 files.
9:58:47 PM: Found 43 cached files for .cache directory with 1 files.
9:58:47 PM: Netlify cache restored
9:58:49 PM: success onPreInit — 4.031 s
9:58:49 PM: success delete html and css files from previous builds — 0.477 s
9:58:50 PM: success initialize cache — 0.151 s
9:58:50 PM: success copy gatsby files — 0.026 s
9:58:50 PM: success onPreBootstrap — 0.008 s
9:58:51 PM: success source and transform nodes — 0.916 s
9:58:51 PM: success building schema — 0.535 s
9:58:53 PM: success createPages — 1.878 s
9:58:53 PM: success createPagesStatefully — 0.319 s
9:58:53 PM: success onPreExtractQueries — 0.003 s
9:58:54 PM: success update schema — 0.369 s
9:58:54 PM: success extract queries from components — 0.669 s
9:59:03 PM: success run graphql queries — 8.741 s — 363/363 41.54 queries/second
9:59:03 PM: success write out page data — 0.013 s
9:59:03 PM: success write out redirect data — 0.001 s
9:59:04 PM: done generating icons for manifest
9:59:04 PM: success onPostBootstrap — 0.803 s
9:59:04 PM: info bootstrap finished - 22.478 s
9:59:49 PM: success Building production JavaScript and CSS bundles — 44.680 s
10:00:05 PM: error Building static HTML for pages failed
10:00:05 PM: See our docs page on debugging HTML builds for help https://goo.gl/yL9lND
10:00:05 PM:   44 |     const styles = headComponents.reduce((str, x) => {
10:00:05 PM:   45 |       if (x.type === "style") {
10:00:05 PM: > 46 |         str += x.props.dangerouslySetInnerHTML.__html;
10:00:05 PM:      | ^
10:00:05 PM:   47 |       } else if (x.key && x.key === "TypographyStyle") {
10:00:05 PM:   48 |         str = `${x.props.typography.toString()}${str}`;
10:00:05 PM:   49 |       }
10:00:05 PM: 
10:00:05 PM:   WebpackError: TypeError: Cannot read property '__html' of undefined
10:00:05 PM:   
10:00:05 PM:   - gatsby-ssr.js:46 headComponents.reduce
10:00:05 PM:     [lib]/[gatsby-plugin-amp]/gatsby-ssr.js:46:1
10:00:05 PM:   
10:00:05 PM:   
10:00:05 PM:   - gatsby-ssr.js:44 Object.onPreRenderHTML
10:00:05 PM:     [lib]/[gatsby-plugin-amp]/gatsby-ssr.js:44:1
10:00:05 PM:   
10:00:05 PM:   - api-runner-ssr.js:52 
10:00:05 PM:     lib/.cache/api-runner-ssr.js:52:39
10:00:05 PM:   
10:00:05 PM:   
10:00:05 PM:   - api-runner-ssr.js:48 ./.cache/api-runner-ssr.js.module.exports
10:00:05 PM:     lib/.cache/api-runner-ssr.js:48:25
10:00:05 PM:   
10:00:05 PM:   - static-entry.js:364 Module../.cache/static-entry.js.__webpack_exports__.defa    ult
10:00:05 PM:     lib/.cache/static-entry.js:364:3
10:00:05 PM:   
10:00:05 PM:   - bootstrap:24 Promise
10:00:05 PM:     lib/webpack/bootstrap:24:1
10:00:05 PM:   
10:00:05 PM:   
10:00:05 PM:   
10:00:05 PM:   - bootstrap:68 new Promise
10:00:05 PM:     lib/webpack/bootstrap:68:1
10:00:05 PM:   
10:00:05 PM:   
10:00:05 PM:   - bootstrap:5 tryCatcher
10:00:05 PM:     lib/webpack/bootstrap:5:1
10:00:05 PM:   
10:00:05 PM:   - bootstrap:50 MappingPromiseArray._promiseFulfilled
10:00:05 PM:     lib/webpack/bootstrap:50:1
10:00:05 PM:   
10:00:05 PM:   - api-runner-ssr.js:6 MappingPromiseArray.PromiseArray._iterate
10:00:05 PM:     lib/.cache/api-runner-ssr.js:6:27
10:00:05 PM:   
10:00:05 PM:   - bootstrap:67 MappingPromiseArray.init
10:00:05 PM:     lib/webpack/bootstrap:67:1
10:00:05 PM:   
10:00:05 PM: 
10:00:06 PM: Caching artifacts
10:00:06 PM: Started saving node modules
10:00:06 PM: Finished saving node modules
10:00:06 PM: Started saving pip cache
10:00:06 PM: Finished saving pip cache
10:00:06 PM: Started saving emacs cask dependencies
10:00:06 PM: Unhandled rejection Error [ERR_IPC_CHANNEL_CLOSED]: channel closed
10:00:06 PM:     at process.target.send (internal/child_process.js:578:16)
10:00:06 PM:     at reportSuccess (/opt/build/repo/node_modules/jest-worker/build/child.js:59:11)
10:00:06 PM:     at tryCatcher (/opt/build/repo/node_modules/bluebird/js/release/util.js:16:23)
10:00:06 PM:     at Promise._settlePromiseFromHandler (/opt/build/repo/node_modules/bluebird/js/release/promise.js:512:31)
10:00:06 PM:     at Promise._settlePromise (/opt/build/repo/node_modules/bluebird/js/release/promise.js:569:18)
10:00:06 PM:     at Promise._settlePromise0 (/opt/build/repo/node_modules/bluebird/js/release/promise.js:614:10)
10:00:06 PM:     at Promise._settlePromises (/opt/build/repo/node_modules/bluebird/js/release/promise.js:694:18)
10:00:06 PM:     at Promise._fulfill (/opt/build/repo/node_modules/bluebird/js/release/promise.js:638:18)
10:00:06 PM:     at MappingPromiseArray.PromiseArray._resolve (/opt/build/repo/node_modules/bluebird/js/release/promise_array.js:126:19)
10:00:06 PM:     at MappingPromiseArray._promiseFulfilled (/opt/build/repo/node_modules/bluebird/js/release/map.js:101:18)
10:00:06 PM:     at Promise._settlePromise (/opt/build/repo/node_modules/bluebird/js/release/promise.js:574:26)
10:00:06 PM:     at Promise._settlePromise0 (/opt/build/repo/node_modules/bluebird/js/release/promise.js:614:10)
10:00:06 PM:     at Promise._settlePromises (/opt/build/repo/node_modules/bluebird/js/release/promise.js:694:18)
10:00:06 PM:     at _drainQueueStep (/opt/build/repo/node_modules/bluebird/js/release/async.js:138:12)
10:00:06 PM:     at _drainQueue (/opt/build/repo/node_modules/bluebird/js/release/async.js:131:9)
10:00:06 PM:     at Async._drainQueues (/opt/build/repo/node_modules/bluebird/js/release/async.js:147:5)
10:00:06 PM: Finished saving emacs cask dependencies
10:00:06 PM: Started saving maven dependencies
10:00:06 PM: Finished saving maven dependencies
10:00:06 PM: Started saving boot dependencies
10:00:07 PM: failed during stage 'building site': Build script returned non-zero exit code: 1
10:00:06 PM: Finished saving boot dependencies
10:00:06 PM: Started saving go dependencies
10:00:07 PM: Shutting down logging, 21 messages pending
```

**Comment by MrRobotjs** [comment]
Followed the steps thoroughly but can’t seem to get this working... Source: https://github.com/MrRobotjs/betterdocs-react/

**Comment by c0b41** [comment]
already fixed #8 wait for the new release

**Comment by jafaircl** [comment]
Just published 0.2.0 so once that populates in the registry this error should be fixed.
